src/index.tsx

```diff
@@ -1055,6 +1162,65 @@ class App extends React.Component<{}, AppState> {
                 return;
               }
 
+              if (isResizingElements && this.state.resizingElement) {
```

Comment:
I'm not sure where the logic breaks but it's possible to get out of sync
![image](https://user-images.githubusercontent.com/197597/71769134-eb5cee00-2ed1-11ea-969f-528d32b82aa7.png)